README.md

```diff
@@ -1 +1,117 @@
-# microsite-generator
\ No newline at end of file
+# microsite-generator
+
+Generates MicroSites for an edX installation.
+
+# Usage
+
+- Copy `config/sample.yaml` to `config/config.yaml`.
+- Check `Makefile` for available commands.
+```
+make help
+
+help:     Show help.
+
+run:  Generate microsite configuration for all services
+run-lms:  Generate microsite configuration in LMS
+run-discovery:  Generate microsite configuration in Discovery
+run-ecommerce:   Generate microsite configuration in eCommerce
+
+dev.run-lms:     Devstack - Generate microsite configuration in LMS
+dev.run-discovery:   Devstack - Generate microsite configuration in Discovery
+dev.run-ecommerce:   Devstack - Generate microsite configuration in eCommerce
+```
+# Structure of config file
```

Comment:
A little more intro on how to set up this project, will go a long way since we might have to tell the client on how to achieve this.

Reply:
Added more documentation on how to install and use it.